pom.xml

```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kie</groupId>
+    <artifactId>jenkins-pipeline-shared-libraries</artifactId>
+    <packaging>jar</packaging>
+    <version>1.0.0</version>
+    <description>Jenkins Pipeline Shared Libraries Testing Project</description>
+
+    <repositories>
+        <repository>
+            <id>jenkins-releases</id>
+            <name>Jenkins Releases</name>
+            <url>http://repo.jenkins-ci.org/releases</url>
+        </repository>
+    </repositories>
+
+    <properties>
+        <groovy.core.version>2.4.11</groovy.core.version>
+        <groovy.gmaven.pluginVersion>1.6.1</groovy.gmaven.pluginVersion>
+        <google.guava.version>20.0</google.guava.version>
+
+        <jenkins-spock.version>2.0.0</jenkins-spock.version>
+        <jenkins.version>2.102</jenkins.version> <!-- this should change to 2.150.2 -->
```

Comment:
Any reason why not changing it straight away? Or do you want to have this merged and upgrade in a separate PR?

Reply:
I forgot to do it, changing it, thanks @winklerm